WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2103,6 +2103,12 @@ static GSourceFuncs _handlerIntervention =
                 browser->OnLoadFinished(Core::ToString(webkit_web_view_get_uri(webView)));
             }
         }
+        static void loadFailedCallback(WebKitWebView*, WebKitLoadEvent loadEvent, const gchar* failingURI, GError* error, WebKitImplementation* browser)
+        {
+            string message(string("{ \"url\": \"") + failingURI + string("\", \"Error message\": \"") + error->message + string("\", \"loadEvent\":") + Core::NumberType<uint32_t>(loadEvent).Text() + string(" }"));
```

Comment:
I think it would be good to match WK, and suppress loadFailed notification of 'canceled' requests (and loadfinished as well)